cloudant-client/src/test/java/com/cloudant/tests/AttachmentsTest.java

```diff
@@ -105,17 +105,20 @@ public void attachmentStandalone() throws IOException, URISyntaxException {
                 .attachmentUri(response.getId(), "foo.txt"));
         InputStream in = clientResource.get().executeRequest(conn).responseAsInputStream();
 
-        ByteArrayOutputStream bytesOut = new ByteArrayOutputStream();
-        int n;
-        while ((n = in.read()) != -1) {
-            bytesOut.write(n);
-        }
-        bytesOut.flush();
-        in.close();
+        try {
+            ByteArrayOutputStream bytesOut = new ByteArrayOutputStream();
```

Comment:
I know these are only test code, and this isn't strictly the domain of this PR, but these tests which read and write 1 byte at a time will be _really_ slow.